src/screens/OtherProfile/index.js

```diff
@@ -380,7 +473,7 @@ const OtherProfile = () => {
       blockUserRef.current.close();
       specificIssueRef.current.close();
       reportUserRef.current.close();
-      refetchBlockStatus();
+      checkUserBlockHandle(dataMain.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `checkUserBlockHandle` function is called after blocking a user. Ensure that this function correctly updates the state and UI to reflect the change in block status.
```diff
+ checkUserBlockHandle(dataMain.user_id);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->